splice_access_api/src/main/java/com/splicemachine/access/configuration/SQLConfiguration.java

```diff
@@ -210,6 +210,9 @@
     public static final String NESTEDLOOPJOIN_BATCH_SIZE = "splice.nestedLoopJoin.batchSize";
     private static final int DEFAULT_NESTEDLOOPJOIN_BATCH_SIZE = 10;
 
+    public static final String CONTROL_EXECUTION_ROWS_LIMIT = "splice.controlExecution.rowsLimit";
+    private static final int DEFAULT_CONTROL_EXECUTION_ROWS_LIMIT = 2000000;
+
```

Comment:
Makes sense to have a smaller size. What would you suggest? 100K rows? I was afraid some of the OLTP queries would end up rejected by this mechanism. Should we disable this if the user forces control execution?